src/common/exception/service.exception.ts

```
@@ -1,9 +1,13 @@
-import { ENTITY_NOT_FOUND, ErrorCode } from './error.code';
+import { ENTITY_NOT_FOUND, ErrorCode, VALIDATION_ERROR } from './error.code';
 
 export const EntityNotFoundException = (message?: string): ServiceException => {
   return new ServiceException(ENTITY_NOT_FOUND, message);
 };
 
+export const ValidationErrorException = (message?: string): ServiceException => {
```

ESLint

[error] Replace `message?:·string` with `⏎··message?:·string,⏎`

(prettier/prettier)